src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -337,10 +338,16 @@ trait BeamHelper extends LazyLogging {
       beamConfig.beam.agentsim.simulationName,
       beamConfig.beam.outputs.addTimestampToOutputDirectory
     )
+
     LoggingUtil.createFileLogger(outputDirectory)
     matsimConfig.controler.setOutputDirectory(outputDirectory)
     matsimConfig.controler().setWritePlansInterval(beamConfig.beam.outputs.writePlansInterval)
 
+    logger.info(s"Starting beam on branch $getBranch at commit $getCommitHash.")
+    val outConf = Paths.get(outputDirectory, "beam.conf")
+    Files.write(outConf, config.root().render(ConfigRenderOptions.concise()).getBytes)
+    logger.info(s"Config [${beamConfig.beam.agentsim.simulationName}] copied to $outConf.")
```

Comment:
What is the point of this? Just makes it easier to debug what was actually configured for that run?

Reply:
to log them in output that is published to s3 for future analyses.